powersoftau/src/batched_accumulator.rs

```diff
@@ -834,21 +850,20 @@ impl<'a, E: Engine + Sync> BatchedAccumulator<'a, E> {
                     check_input_for_correctness,
                     &input,
                 )?;
-                assert_eq!(
-                    accumulator.tau_powers_g2.len(),
-                    0,
-                    "during rest of tau g1 generation tau g2 must be empty"
-                );
+                if !accumulator.tau_powers_g2.is_empty() {
+                    return Err(Error::InvalidLength {
+                        expected: 0,
+                        got: accumulator.tau_powers_g2.len(),
+                    });
+                }
 
                 let taupowers = generate_powers_of_tau::<E>(&key.tau, start, size);
                 batch_exp(&mut accumulator.tau_powers_g1, &taupowers[0..], None);
-                //accumulator.beta_g2 = accumulator.beta_g2.mul(key.beta).into_affine();
```

Comment:
Was there any point to having this code?

Reply:
no, it was just wrong tbh.